src/config/graphical-session/kde.md

```diff
@@ -17,3 +17,23 @@ instead.
 If you wish to start an X-based session from the console, use
 [startx](./xorg.md#startx) to run `startplasma-x11`. For a Wayland-based
 session, run `startplasma-wayland` directly.
+
+## Dolphin
+
+Dolphin is the default file manager of the KDE desktop environment. It can be
+installed on its own by installing the `dolphin` package, or it can be installed
+as part of the `kde5` or `kde5-baseapps` packages.
+
+### Thumbnail Previews
+
+To have properly working thumbnail file previews, the packages
+`ffmpegthumbnailer` and `kdegraphics-thumbnailers` have to be installed. Enable
```

Comment:
I'm curious why `ffmpegthumbnailer` isn't a dependency for the kde graphics program. Would you mind opening an issue (or better yet, PR) for it in void-packages?

Reply:
Sure thing, just opened a PR: https://github.com/void-linux/void-packages/pull/23936

Reply:
Great, thanks! Might make sense to ping th  maintainer or Johnnynator there.